README.md

```diff
@@ -337,6 +337,21 @@ INVALID_EMAIL,bob
 
 The error returned should be: `{ "E-Mail" => "is invalid" }`
 
+### Complex quoting (Illegal quoting problem)
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR")
+  # => CSV::MalformedCSVError: Illegal quoting in line 1.
+```
+
+The problem is that the CSV comma-delimited text should entirely surround by `"`
+not in part or do not contain `"` at all. You can pass `quote_char` option.
+
+```ruby
+CSV.parse("1,Glk,Lev \"The Black Spider\" Yashin,USSR", quote_char: "\x00")
+  # => [ ["1", "Glk", "Lev \"The Black Spider\" Yashin", "USSR"] ]
```

Comment:
Got it. This solution works with quote_char: "|", quote_char: "@" and any char that a CSV file not contains. It seems like '\x00' char is a proven option, but I can use single quote in the example.

Absolutely, I should use CSVImporter in the example, miss that point.